src/test/javascript/spec/component/exam/participate/exam-participation.component.spec.ts

```diff
@@ -885,44 +884,6 @@ describe('ExamParticipationComponent', () => {
     });
 
     describe('handleHandInEarly', () => {
-        it('should reset pageComponentVisited after the hand-in-early window is closed', () => {
```

Comment:
If I understand correctly this case is not tested anymore?

Reply:
yes, it is not needed anymore